Sources/Features/Attributes/AttributesProvider.swift

```diff
@@ -39,6 +39,10 @@ extension AttributesProvider: SignalContext {
         self.faultInfo.faultMessage = faultMessage
     }
     
+    func setErrorType(type: String?) {
```

Comment:
Why do we need this field now?

Reply:
We're trying to followup conventions from other libraries to tell user about error type - in this case we set error type for crashes (error.type = crash) and oom - (low memory)